src/openzaak/components/zaken/openapi.yaml

```diff
@@ -9415,8 +8789,6 @@ components:
           description: URL-referentie naar de EIGENSCHAP (in de Catalogi API).
           type: string
           format: uri
-          maxLength: 1000
-          minLength: 1
```

Comment:
This should be back and hopefully with a generic solution.

Reply:
please take a look at https://github.com/open-zaak/open-zaak/pull/51#discussion_r315630538